data/src/main/java/com/globant/data/repositories/PoseAngleDetectorImpl.kt

```diff
@@ -98,22 +59,16 @@ class PoseAngleDetectorImpl(
 
     private fun getAngle(
         pose: Pose,
-        firstLandmarkType: PoseLandmarkType,
-        midLandmarkType: PoseLandmarkType,
-        lastLandmarkType: PoseLandmarkType
+        jointAngle: JointAngle
     ): Double {
-        return requireNotNull(
-            pose?.let { pose ->
-                val firstLandmark = firstLandmarkType.toPoseLandmark(pose)
-                val midLandmark = midLandmarkType.toPoseLandmark(pose)
-                val lastLandmark = lastLandmarkType.toPoseLandmark(pose)
+        val firstLandmark = jointAngle.firstLandmarkType.toPoseLandmark(pose)
+        val midLandmark = jointAngle.midLandmarkType.toPoseLandmark(pose)
+        val lastLandmark = jointAngle.lastLandmarkType.toPoseLandmark(pose)
 
-                getAngleBetweenPoseLandmark(
-                    requireNotNull(firstLandmark),
-                    requireNotNull(midLandmark),
-                    requireNotNull(lastLandmark)
-                )
-            }
+        return getAngleBetweenPoseLandmark(
+            requireNotNull(firstLandmark),
+            requireNotNull(midLandmark),
+            requireNotNull(lastLandmark)
         )
     }
 
```

Comment:
Please replace `getPosition()` in the lines below by the usage of `position` property access. Also, replace Java Math functions by Kotlin Math functions.